runtime/src/accounts_db.rs

```diff
@@ -3057,15 +3057,15 @@ impl AccountsDb {
             }
         }
         measure.stop();
-        inc_new_counter_info!(
+        inc_new_counter_debug!(
             "shrink_select_top_sparse_storage_entries-ms",
             measure.as_ms() as usize
         );
-        inc_new_counter_info!(
+        inc_new_counter_debug!(
             "shrink_select_top_sparse_storage_entries-seeds",
             candidates_count
         );
-        inc_new_counter_info!(
+        inc_new_counter_debug!(
```

Comment:
@lijunwangs these are becoming spammy and will need to be aggregated if they want to be kept around, any objections?

Reply:
no problem

Reply:
https://github.com/solana-labs/solana/issues/24457
@lijunwangs do you mind to take a look into this?

Reply:
I think this is okay to turn them into inc_new_counter_debug! I do not expect we are going to see a lot of insight into this any longer.